src/ArrayOfSets.hpp

```diff
@@ -422,13 +422,15 @@ class ArrayOfSets : protected ArrayOfSetsView< T, INDEX_TYPE >
 
     /**
      * @brief Callback signaling that the size of the set has increased.
+     * @param [in] curPtr the current pointer to the array.
      * @param [in] nToAdd the increase in the size.
      * @note This method doesn't actually change the size, it just checks if the new size
      *       exceeds the capacity of the set and if so reserves more space.
      * @return a pointer to the sets values.
      */
     inline
-    T * incrementSize( INDEX_TYPE const nToAdd ) const LVARRAY_RESTRICT_THIS
+    T * incrementSize( T * const LVARRAY_UNUSED_ARG( curPtr ),
```

Comment:
can you change `LVARRAY_UNUSED_ARG` to `LVARRAY_UNUSED_PARAM`?...
and why add this parameter if it us unused?

Reply:
It's used in the default `sortedArrayManipulation::CallBack::incrementSize`.